cross/jellyfin-web/Makefile

```diff
@@ -23,4 +23,4 @@ PATH := $(WORK_DIR)/../../../native/nodejs/work-native/node/bin:$(PATH)
 jellyfin_install:
 	$(RUN) npm install && npm run build:production
 	install -m 755 -d $(STAGING_INSTALL_PREFIX)/web
-	cp -R $(WORK_DIR)/$(PKG_DIR)/dist/* $(STAGING_INSTALL_PREFIX)/web/
+	cp -R $(WORK_DIR)/$(PKG_DIR)/dist/* $(STAGING_INSTALL_PREFIX)/web/
```

Comment:
please revert this change (keep the final line break)

Reply:
Already solved